src/constants/routes.ts

```diff
@@ -52,7 +51,12 @@ export const Paths = {
   portfolio: PORTFOLIO_ROUTE,
   orders: '/orders',
   order: ORDER_ROUTE,
-  approval: APPROVAL_ROUTE
+  approval: '/approval/requests',
+  requests: '/approval/requests',
```

Comment:
Are these meant to be the same value?

Reply:
yes - it's the default path for approval